packages/phoenix-event-display/src/tests/loaders/phoenix-loader.test.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @jest-environment jsdom
+ */
+import { InfoLogger } from '../../helpers/info-logger';
+import { PhoenixLoader } from '../../loaders/phoenix-loader';
+import { ThreeManager } from '../../managers/three-manager/index';
+import { UIManager } from '../../managers/ui-manager/index';
+
+jest.mock('../../managers/three-manager/index');
+
+describe('PhoenixLoader', () => {
+  let phoenixLoader: PhoenixLoader;
+
+  beforeEach(() => {
+    phoenixLoader = new PhoenixLoader();
+  });
+
+  it('should create an instance', () => {
+    expect(phoenixLoader).toBeTruthy();
+  });
+
+  it('should take an object that represents ONE event and takes care of adding', () => {
+    const eventData = {};
+    const graphicsLibrary = new ThreeManager(new InfoLogger());
+    const uiManager = new UIManager(new ThreeManager(new InfoLogger()));
+    const infoLogger = new InfoLogger();
+
+    jest.spyOn(phoenixLoader, 'buildEventData');
+    phoenixLoader.buildEventData(
+      eventData,
+      graphicsLibrary,
+      uiManager,
+      infoLogger
+    );
+    expect(phoenixLoader.buildEventData).toHaveBeenCalled();
```

Comment:
I think this assertion is redundant. We are calling the function in the previous statement and just checking here if it's called. We should test if it actually loads the object types specified in the event data.

Reply:
I have the same comments for all the other test cases in this suite. We are not testing anything here. We need to assert that the correct methods are called internally in these functions.

And if the functions return anything, that return value should be asserted.